example/spec/parallel/startSF/startSfSpec.js

```diff
@@ -199,72 +211,24 @@ describe('the sf-starter lambda function', () => {
       }).promise();
       maxQueueUrl = QueueUrl;
 
-      templateKey = `${config.stackName}/workflows/${waitPassSfName}.json`;
-      templateUri = `s3://${config.bucket}/${templateKey}`;
-
-      console.log('expected template URI', templateUri);
-
       const { stateMachineArn } = await sfn().createStateMachine(waitPassSfParams).promise();
       waitPassSfArn = stateMachineArn;
 
-      console.log(`expected waitPass state machine ARN: ${waitPassSfArn}`);
-
-      await s3PutObject({
-        Bucket: config.bucket,
-        Key: templateKey,
-        Body: JSON.stringify({
-          cumulus_meta: {
-            state_machine: waitPassSfArn
-          },
-          meta: {
-            queues: {
-              [maxQueueName]: maxQueueUrl
-            },
-            queueExecutionLimits: {
-              [maxQueueName]: queueMaxExecutions
-            }
-          }
-        })
+      await sendStartSfMessages({
+        numOfMessages: totalNumMessages,
+        queueMaxExecutions,
+        queueName: maxQueueName,
+        queueUrl: maxQueueUrl,
+        workflowArn: waitPassSfArn
       });
-
-      await Promise.all([
-        // postRule({
-        //   prefix: config.stackName,
-        //   rule: {
-        //     name: ruleName,
-        //     workflow: waitPassSfName,
-        //     collection: {
-        //       name: collection.name,
-        //       version: collection.version
-        //     },
-        //     state: 'ENABLED',
-        //     rule: {
-        //       type: 'onetime'
-        //     }
-        //   }
-        // }),
-        addCollections(config.stackName, config.bucket, collectionsDir, testSuffix)
-      ]);
     });
 
     afterAll(async () => {
-      const deleteStateMachine = doStateMachineDelete ?
-        sfn().deleteStateMachine({ stateMachineArn: waitPassSfArn }).promise() :
-        Promise.resolve();
-
-      // Have to delete rule before associated collection
-      // await deleteRule({
-      //   prefix: config.stackName,
-      //   ruleName
-      // });
-
       await Promise.all([
-        deleteS3Object(config.bucket, templateKey),
         sqs().deleteQueue({
           QueueUrl: maxQueueUrl
         }).promise(),
-        deleteCollections(config.stackName, config.bucket, [collection]),
-        deleteStateMachine,
+        sfn().deleteStateMachine({ stateMachineArn: waitPassSfArn }).promise(),
         dynamodbDocClient().delete({
```

Comment:
This does delete the semaphore record, but because this test doesn't wait for the executions to finish, once they do finish they call `semaphore.down()`, which will always create the record if it doesn't exist before decrementing. Without making the test wait for all executions to finish, which seems wasteful, I'm not sure what to do here, but I don't love that the test doesn't fully clean up after itself.